metabase/metabase
# Search Milestone 2: Add `verified` search filter

_**Note: this PR is blocked until the `Switch` Mantine component is available in the very near future.** Otherwise, the code is ready to be reviewed so have at it_ 😄

`verified` filter for Search Milestone 2: #27982 

### Description

Adds `verified` filter toggle to the search sidebar. The code only appears when using enterprise with the `content_verification` token enabled.


### How to verify

Start an enterprise instance with `yarn dev-ee` - ensure that you have an enterprise token (instructions are [here](https://www.notion.so/metabase/Cypress-How-To-4d1c70bb0ce94a60b3688fd7acba8471?pvs=4))

Go to the search sidebar on an enterprise instance, and filter by verified items only. Only verified assets should be available in the search after the filter.

After filtering, try to refresh the page or paste the URL into a new window to test that hydrating the search filters still works when entering the search directly from a URL. The results should remain the same and should behave in the same way.

### Tests
